src/main/java/org/repodriller/scm/CommitVisitor.java

```diff
@@ -20,6 +20,8 @@
 import org.repodriller.persistence.PersistenceMechanism;
 
 public interface CommitVisitor {
+	default void initialize(SCMRepository repo, PersistenceMechanism writer) {}
```

Comment:
nice! much cleaner than my `RequiresBlaBla` interface. I completely forgot about java 8 default methods in interfaces!